src/cache/edonly_edmf_precomputed_quantities.jl

```diff
@@ -0,0 +1,119 @@
+"""
+    set_edonly_edmf_precomputed_quantities_closures!(Y, p, t)
+
+Updates the precomputed quantities stored in `p` for edmfx closures.
+"""
+NVTX.@annotate function set_edonly_edmf_precomputed_quantities_closures!(
+    Y,
+    p,
+    t,
+)
+    (; moisture_model, turbconv_model) = p.atmos
+    @assert !(moisture_model isa DryModel)
+
+    FT = Spaces.undertype(axes(Y.c))
+
+    thermo_params = CAP.thermodynamics_params(p.params)
+
+    (;
+        ᶜmixing_length_tuple,
+        ᶜmixing_length,
+        ᶜlinear_buoygrad,
+        ᶜstrain_rate_norm,
+        ᶜK_u,
+        ᶜK_h,
+        ᶜstrain_rate_norm,
+        ᶜts,
+        ᶠu³,
+        ᶜtke⁰,
+        ρatke_flux,
+    ) = p.precomputed
+    (; ᶜgradᵥ_θ_virt, ᶜgradᵥ_q_tot, ᶜgradᵥ_θ_liq_ice, ᶜspecific) = p.precomputed
+    (; ustar, obukhov_length, buoyancy_flux) = p.precomputed.sfc_conditions
+
+    @. ᶜgradᵥ_θ_virt = ᶜgradᵥ(ᶠinterp(TD.virtual_pottemp(thermo_params, ᶜts)))       # ∂θv∂z_unsat
+    @. ᶜgradᵥ_q_tot = ᶜgradᵥ(ᶠinterp(ᶜspecific.q_tot))                                        # ∂qt∂z_sat
+    @. ᶜgradᵥ_θ_liq_ice =
```

Comment:
This shouldn't be needed because they are calculated here already: https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/cache/precomputed_quantities.jl#L415 (although I don't know whether that's the best place to calculate them).